node/tests/SharedTests.ts

```diff
@@ -5743,6 +5743,156 @@ export function runBaseTests<Context>(config: {
+                    expect(
+                        await client.restore(key2, 0, data, { replace: true }),
+                    ).toEqual("OK");
+
+                    // Restore with `REPLACE`, `ABSTTL`, and positive TTL
+                    expect(
+                        await client.restore(key2, 1000, data, {
+                            replace: true,
+                            absttl: true,
+                        }),
+                    ).toEqual("OK");
+
+                    // Restore with `REPLACE`, `ABSTTL`, and negative TTL
+                    try {
+                        expect(
+                            await client.restore(key2, -10, data, {
+                                replace: true,
+                                absttl: true,
+                            }),
+                        ).toThrow();
+                    } catch (e) {
+                        expect((e as Error).message).toMatch(
+                            "ResponseError: Invalid TTL value",
+                        );
+                    }
+
+                    // Restore with REPLACE and positive idletime
+                    expect(
+                        await client.restore(key2, 0, data, {
+                            replace: true,
+                            idletime: 10,
+                        }),
+                    ).toEqual("OK");
+
+                    // Restore with REPLACE and negative idletime
+                    try {
+                        expect(
+                            await client.restore(key2, 0, data, {
+                                replace: true,
+                                idletime: -10,
+                            }),
+                        ).toThrow();
+                    } catch (e) {
+                        expect((e as Error).message).toMatch(
+                            "ResponseError: Invalid IDLETIME value",
+                        );
+                    }
+
+                    // Restore with REPLACE and positive frequency
+                    expect(
+                        await client.restore(key2, 0, data, {
+                            replace: true,
+                            frequency: 10,
+                        }),
+                    ).toEqual("OK");
+
+                    // Restore with REPLACE and negative frequency
+                    try {
+                        expect(
+                            await client.restore(key2, 0, data, {
+                                replace: true,
+                                frequency: -10,
+                            }),
+                        ).toThrow();
+                    } catch (e) {
+                        expect((e as Error).message).toMatch(
+                            "ResponseError: Invalid FREQ value",
+                        );
+                    }
+
+                    // Restore only uses IDLETIME or FREQ modifiers
+                    // Error will be raised if both options are set
```

Comment:
Tell in doc that they can't be set both. Do we need to update docs in other clients too?